src/test/tests/databases/silo.py

```diff
@@ -98,6 +98,8 @@ def TurnOffSetsByName(silr, cat, names):
             if setname == names:
                 silr.TurnOffSet(s)
 
+if False: '''
```

Comment:
Did you mean to leave this in ?

Reply:
Whoops! 😮 Thanks for pointing out. No, I didn't mean to leave that in. Fixed now.